src/components/Tooltip/hooks/useEnterLeaveHandlers.ts

```diff
@@ -0,0 +1,26 @@
+import React, { useCallback, useState } from 'react';
+
+export const useEnterLeaveHandlers = <T extends HTMLElement>({
+  onMouseEnter,
+  onMouseLeave,
+}: Partial<Record<'onMouseEnter' | 'onMouseLeave', React.MouseEventHandler<T>>> = {}) => {
+  const [popperVisible, setPopperVisible] = useState(false);
```

Comment:
I would not give names according to a library that can change over time. Could we instead use name like `tooltipVisible`?